markwiki/application.py

```diff
@@ -26,6 +27,11 @@ def build_app(app_name):
     user_storage_factory = UserStorageFactory()
     app.user_storage = user_storage_factory.get_storage(app.config)
 
+    # init git
```

Comment:
I don't think this comment helps. Your code is very clear here! :smile: Please remove this comment.